io-onboarding-pa-api/values.yaml

```diff
@@ -46,6 +46,12 @@ ioOnboardingPaApi:
     saml_cert_path: "/usr/src/app/certs/cert.pem"
     saml_key_path: "/usr/src/app/certs/key.pem"
     token_duration_in_seconds: "3600" #1 hour
+    email_password: "$onboarding1234$"
```

Comment:
please, IMMEDIATELY change the password and send the new one privately to me. I'll add it to the keyvault.
Also, remove this line from here